packages/types/src/clerk.ts

```diff
@@ -772,6 +779,7 @@ export type ClerkOptions = ClerkOptionsNavigation &
       {
         persistClient: boolean;
         rethrowOfflineNetworkErrors: boolean;
+        withSessionTasks: boolean;
       },
```

Comment:
This is only being introduced temporarily to not affect the bundle size once this PR gets merged 

We're working on FAPI changes to indicate that tasks are enabled on a instance by looking at the environment response, and so the SDK will only load those components based on that

Reply:
Let's add a comment above the option to explain what it's for, so we know to remove later.